src/NzbDrone.Core/Notifications/Email/EmailService.cs

```diff
@@ -54,13 +49,20 @@ public void SendEmail(EmailSettings settings, string subject, string body, bool
             }
         }
 
-        private void Send(MailMessage email, string server, int port, bool ssl, NetworkCredential credentials)
+        private void Send(MimeMessage email, string server, int port, string user = null, string password = null)
```

Comment:
Is there ever a case where `user` and `password` aren't explicitly passed in? If not they don't need default values.

Reply:
This goes away with the above change